spaceship/lib/spaceship/connect_api/testflight/testflight.rb

```diff
@@ -412,6 +412,15 @@ def get_beta_tester_metrics(filter: {}, includes: nil, limit: nil, sort: nil)
           test_flight_request_client.get("betaTesterMetrics", params)
         end
 
+        #
+        # buildBundles
+        #
+
+        def get_build_bundles_build_bundle_file_sizes(build_bundle_id:, limit: nil)
```

Comment:
This name is quite the mouthful 😄 

I stuck with it because the endpoint is `buildBundles/{id}/buildBundleFileSizes` so there is some repetition here already... There are other buildBundles methods that don't have the repetition, so if we implemented everything, I was imagining that it would look like this:

Method|Endpoint
---|---
`get_build_bundles_build_bundle_file_sizes`|`buildBundles/{id}/buildBundleFileSizes`
`get_build_bundles_ app_clip_domain_cache_status`|`buildBundles/{id}/appClipDomainCacheStatus`
`get_build_bundles_app_clip_domain_debug_status`|`buildBundles/{id}/appClipDomainDebugStatus`
`get_build_bundles_beta_app_clip_invocations`|`buildBundles/{id}/betaAppClipInvocations`

Would that make sense? If there is a better approach, please let me know 🙏 I couldn't really find any other examples to follow.